provider/hybrid.go

```diff
@@ -35,8 +35,7 @@ const dockerImageTok = "docker:index/image:Image"
 // gRPC methods for the hybrid provider
 
 func (dp dockerHybridProvider) Attach(ctx context.Context, attach *rpc.PluginAttach) (*emptypb.Empty, error) {
-	//TODO implement me
-	panic("implement me")
+	return dp.nativeProvider.Attach(ctx, attach)
```

Comment:
Is there a reason we don't also attach to the bridged provider?

Reply:
It didn't occur to me that one could attach to two providers :) Fixed.